### PR TITLE
i18n(hi): update `getting-started` translation

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description
- translate `getting-started.mdx` for hindi language
- update sidebar ui for couple of hindi texts

Edit: Adding Preview Link for the changes: [Link](https://starlight-gzja3f3g5-astrodotbuild.vercel.app/hi/getting-started/)

### CODE REVIEW

File: docs/src/content/docs/hi/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,173 @@
+---
+title: पहले कदम
+description: जानें कि Astro द्वारा Starlight के साथ अपना अगला अगली दस्तावेज़ीकरण साइट का निर्माण कैसे शुरू करें।
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Starlight [Astro](https://astro.build) फ्रेमवर्क के शीर्ष पर निर्मित एक पूर्ण-विशेषीकृत दस्तावेज़ीकरण थीम है।
+यह मार्गदर्शिका आपको एक नई परियोजना शुरू करने में मदद करेगी।
+मौजूदा Astro परियोजना में Starlight जोड़ने के लिए [मैन्युअल सेटअप निर्देश](/manual-setup/) देखें।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/getting-started.mdx:10]:
```suggestion
मौजूदा Astro परियोजना में Starlight जोड़ने के लिए [मैन्युअल सेटअप निर्देश](/hi/manual-setup/) देखें।
```

This should point to the translated version even if it does not exist yet.

File: docs/src/content/docs/hi/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+अपनी साइट ब्राउज़ करने के लिए इस URL को खोलें।
+
+### सामग्री जोड़ें
+
+Starlight आपके लिए नई सामग्री जोड़ने, या अपनी मौजूदा फ़ाइलें लाने के लिए तैयार है!
+
+#### फ़ाइल स्वरूप
+
+Starlight बिना किसी कॉन्फ़िगरेशन की आवश्यकता के Markdown और MDX में सामग्री लिखने का समर्थन करता है।
+आप प्रायोगिक [Astro Markdoc एकीकरण](https://docs.astro.build/en/guides/integrations-guide/markdoc/) स्थापित करके Markdoc के लिए समर्थन जोड़ सकते हैं।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/getting-started.mdx:89]:
```suggestion
आप प्रायोगिक [Astro Markdoc एकीकरण](https://docs.astro.build/hi/guides/integrations-guide/markdoc/) स्थापित करके Markdoc के लिए समर्थन जोड़ सकते हैं।
```

File: docs/src/content/docs/hi/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+```
+
+यदि आप कुछ भी महत्वपूर्ण भूल जाते हैं, तो Starlight आपको बता देगा।
+
+### अगले कदम
+
+- **कॉन्फ़िगर:** ["कस्टमाइज़िंग स्टारलाइट"](/hi/guides/customization/) में सामान्य विकल्पों के बारे में जानें।
+- **नेविगेट:** ["साइडबार नेविगेशन"](/hi/guides/sidebar/) गाइड के साथ अपना साइडबार सेट करें।
+- **अवयवों:** ["अवयवों"](/hi/guides/components/) मार्गदर्शिका में अंतर्निहित कार्ड, टैब और बहुत कुछ खोजें।
+- **डिप्लॉय:** Astro Docs में ["अपनी साइट परिनियोजित करें"](https://docs.astro.build/en/guides/deploy/) मार्गदर्शिका के साथ अपना कार्य प्रकाशित करें।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/getting-started.mdx:129]:
```suggestion
- **डिप्लॉय:** Astro Docs में ["अपनी साइट परिनियोजित करें"](https://docs.astro.build/hi/guides/deploy/) मार्गदर्शिका के साथ अपना कार्य प्रकाशित करें।
```

File: docs/src/content/docs/hi/getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+```
+
+</TabItem>
+</Tabs>
+
+प्रत्येक रिलीज़ में किए गए परिवर्तनों की पूरी सूची के लिए [Starlight चेंजलॉग](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) देखें।
+
+## समस्या निवारण Starlight में
+
+यह सुनिश्चित करने के लिए कि आपकी Starlight साइट कॉन्फ़िगर है और ठीक से काम कर रही है, [परियोजना कॉन्फ़िगरेशन](/reference/configuration/) और [व्यक्तिगत पेज फ्रंटमैटर कॉन्फ़िगरेशन](/reference/frontmatter/) संदर्भ पेजो का उपयोग करें।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/getting-started.mdx:167]:
```suggestion
यह सुनिश्चित करने के लिए कि आपकी Starlight साइट कॉन्फ़िगर है और ठीक से काम कर रही है, [परियोजना कॉन्फ़िगरेशन](/hi/reference/configuration/) और [व्यक्तिगत पेज फ्रंटमैटर कॉन्फ़िगरेशन](/hi/reference/frontmatter/) संदर्भ पेजो का उपयोग करें।
```

File: docs/src/content/docs/hi/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+सामग्री जोड़ने और अपनी स्टारलाइट साइट को अनुकूलित करने में सहायता के लिए साइडबार में मार्गदर्शिकाएँ देखें।
+
+यदि आपका उत्तर इन दस्तावेज़ों में नहीं मिल सकता है, तो संपूर्ण Astro दस्तावेज़ीकरण के लिए कृपया [Astro दस्तावेज़](https://docs.astro.build) देखें।
```
Comment (HiDeoo) [inline docs/src/content/docs/hi/getting-started.mdx:170]:
```suggestion
यदि आपका उत्तर इन दस्तावेज़ों में नहीं मिल सकता है, तो संपूर्ण Astro दस्तावेज़ीकरण के लिए कृपया [Astro दस्तावेज़](https://docs.astro.build/hi) देखें।
```

**Comment by at-the-vr** [comment]
> Thanks for the translation 🙌
> 
> As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and everything looks pretty good to me 👍
> 
> I suggested a few changes to links so they always point to translated pages (even if they do not exist yet) but these are pretty minor changes.

Links updated for hindi pages👍

**Review by delucis — APPROVED** [review]
Looks great! Thanks for translating @at-the-vr and reviewing @HiDeoo 🚀

**Pull request merged**